README.md

```diff
@@ -1,6 +1,50 @@
 [![Community Project header](https://github.com/newrelic/opensource-website/raw/master/src/images/categories/Community_Project.png)](https://opensource.newrelic.com/oss-category/#community-project)
 
-# docs-website
+# New Relic Docs
+
+This repo contains the documentation website code and Markdown source files for  our preview Docs site: docs-preview.newrelic.com. When it's ready, all of our docs will live here.
```

Comment:
I think a bit more context on what this is would be helpful. Which site does this represent, what is the relationship between the repo and the site.

Tiny thing: There are two spaces between `for  our`

Reply:
I keep getting burned by this two spaces thing on the line breaks. Something about how VSCode handles this. I keep an eye out for them.

Good suggestions!